components/AppFooter/AppFooter.tsx

```diff
@@ -0,0 +1,164 @@
+import { createStyles, Text, Container, ActionIcon, Group } from '@mantine/core';
+import AppLink from '../AppLink/AppLink';
+import Image from "next/image";
+import LogoSmall from "../../public/logo_small.png";
+import { DiscordIcon } from "../AppHeader/DiscordIcon";
+import { SteamIcon } from "./SteamIcon";
+import { FooterTextLinks } from "./FooterTextLinks"
+
+const useStyles = createStyles((theme) => ({
+  footer: {
+    marginTop: 120,
+    paddingTop: theme.spacing.xl * 2,
+    paddingBottom: theme.spacing.xl * 2,
+    backgroundColor: theme.colorScheme === 'dark' ? theme.colors.dark[6] : theme.colors.gray[0],
```

Comment:
There is no theme toggle, thus, we don't need to check for `colorScheme`.
Happens multiple times in this file.